tests/scores/test_continuous.py

```diff
@@ -39,6 +40,34 @@ def test_mse_pandas_series():
     result = scores.continuous.mse(fcst_pd_series, obs_pd_series)
     assert round(result, 4) == expected
 
+def test_pandas_series_preserve():
+    """
+    Test calculation works correctly on pandas series
+    """
```

Comment:
How dimension handling should be done for pandas is ... interesting. For the Series specifically, it is inherently a one-dimensional data type (even if the elements have a length), so preserving and conserving don't have an effect. So it's more like preserve/reduce doesn't specifically break for series. For a dataframe, however, they could have multiple dimensions, and preservation and reduction are not implemented for dataframes. It's worthy of consideration, but a lot of use cases will be supported through series/columns, rather than needing to pass in a full dataframe.

I have tried to extend the implementation just one step in this MR. I'll raise a separate issue for dataframe handling. I have clarified that dimensional reduction isn't supported for pandas dataframes.